typespec/UCP/examples/2023-10-01-preview/AWSCredential_AccessKey_Delete.json

```diff
@@ -0,0 +1,14 @@
+{
+    "operationId": "AwsCredentials_Delete",
+    "title": "Delete an AWS AccessKey credential",
+    "parameters": {
+        "api-version": "2023-10-01-preview",
+        "planeType": "aws",
+        "planeName": "awscloud",
+        "credentialName": "default"
```

Comment:
I didn't realize we don't require credential kind for delete and get.. so the uniqueness is based on name and not name + type?

Reply:
yes. For now its based on just name, since we support only one credential that works. But we would have to revisit this as part of supporting multiple credentials in future.